### PR TITLE
GitHub Actions should run automated WordPress tests

### PR DESCRIPTION
**Pull request by kjohnson**
## Description

Adds a workflow for executing automated testing in a GitHub Action.

## Affects

Updates the legacy phpunit test suite.

## Pre-review Checklist

-   [x] Relevant `@since` tags included in DocBlocks
-   [x] Changes logged to the `Unreleased` section of `CHANGELOG.md`
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: tests/unit-tests/tests-give.php [hunk 1]
```diff
@@ -15,13 +15,6 @@ public function tearDown() {
 		parent::tearDown();
 	}
 
-	/**
-	 * @covers Give
-	 */
-	public function test_give_instance() {
-		$this->assertClassHasStaticAttribute( '_instance', 'Give' );
```
Comment (kjohnson) [inline tests/unit-tests/tests-give.php:22]: With the refactor to a Service Container, the static instance has been moved out of the Give class.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+
+-   Automated unit and integrations tests are now executing (#5489)
```
Comment (JasonTheAdams) [inline CHANGELOG.md:12]: Please add a line under `### Changed` to indicate the move from Travis to Github Actions.

**Review by JasonTheAdams — APPROVED** [review]
This is great! Thanks for cleaning this up and getting things working again! I added one comment for the changelog, but this is otherwise good to go. 👍

File: CHANGELOG.md (hunk 2 above)
Comment (kjohnson) [reply CHANGELOG.md:12]: Ah, yeah. That makes sense. Good idea.

**Pull request merged**